content/schedule/activities.md

```diff
@@ -9,4 +9,68 @@ people: all
+
+All activities take place between 15:00 and 18:00, though museums listed below **close at 17:00**.
+
+
+Please note that these excursions are **self-guided**, allowing you to visit your chosen activity at your own pace. All activities are within a 30-minute walking distance of the contest site. Dinner is served in the Flux building from 18:00 until 20:00, so be sure to come by on time after your activity.
+
+The registration form is available through a link in the information email sent on 02/11/2024.
+
+The following options are available:
+
+---
+
+- Museum Prinsenhof
+
+Explore the rich history of Delft and the Netherlands at Museum Prinsenhof [(Google Maps link)](https://maps.app.goo.gl/JJdyDBYoMFUKbu6N7), where you’ll learn about William of Orange and his role in the Dutch struggle for independence. The museum, located in a former monastery, offers a captivating view into Delft's role in Dutch art and culture. 
+
+---
+
+- Museum Royal Delft
+
+Visit the Royal Delft Museum [(Google Maps link)](https://maps.app.goo.gl/PmE3X8AzZPQS2qj19), home to the iconic Delft Blue pottery, crafted here since the 17th century. Enjoy a walk around the museum, admire beautiful hand-painted ceramics, and witness artisans at work, preserving this cherished Dutch tradition.
+
+---
+
+- Vermeer Centrum Delft
+
+Dive into the world of Johannes Vermeer at Vermeer Centrum Delft [(Google Maps link)](https://maps.app.goo.gl/Zjb6iyfE6Da3Q97s5), where you’ll find reproductions of his masterpieces and insights into his life in Delft. This experience lets you uncover the techniques and themes of Delft’s most famous painter. 
+
+---
+
+- New Church Visit
+
+Enjoy free entrance to the iconic 'New Church' in the vibrant city center of Delft [(Google Maps link)](https://maps.app.goo.gl/WdHhoEGKJjMLavBf8). This beautiful piece of architecture is the tallest building in Delft.
+
+---
+
+- Beer Tasting in Local Delft Brewery
+
+You will visit the local Delft brewery 'de Koperen Kat', or, in English, 'the Copper Cat', [(Google Maps link)](https://goo.gl/maps/eXBfGVkCqJzyyMmK9) and sample a selection of unique, locally crafted beers.
+
+---
+
+- Other activity
+
+Take matters into your own hands and embark on an adventure in Delft! You are free to do what you like during activities time. We provide an overview of suggestions for other activities [on this map](https://wisv.ch/map). Note that all 'other activities' are self-paid and on your own accord. We do **not** reimburse expenses made.
+
+---
+
+ - List of other activites (details [on this map](https://wisv.ch/map)):
+
+   - [TU Delft Hortus Botanicus](https://maps.app.goo.gl/ww5hBAZdbBbtsTx1A)
+   - [Film House Lumen](https://maps.app.goo.gl/LQAk2A5bHA9TmqvKA)
+   - [Handicraft Museum Mensert](https://maps.app.goo.gl/hWhi582puS8a5jbPA)
+   - [Museum Prinsenhof Delft](https://maps.app.goo.gl/UqdMJwhu3m2LGUZi8)
+   - [Vermeer Museum Delft](https://maps.app.goo.gl/bPupQDNjTC5ipLqZ7)
```

Comment:
shouldn't these two been removed since we include these this time?

Reply:
correct, I already removed those on the wisv.ch/map but forgot to remove them here. thanks!